src/components/Fetch/search.spec.js

```diff
@@ -38,7 +37,11 @@ test('Fetch-Search', function() {
       colorMap={{}}
     />
   );
+  return wrapper;
+}
 
+test('Fetch-Search', function() {
+  const wrapper = makeWrapper();
   // Testing default state, no entry for searchbar
   assert.equal(wrapper.state('findIdx'), -1);
```

Comment:
Can you move these asserts into makeWrapper, and delete the commented line below.